src/codecs/processor-worker.ts

```diff
@@ -0,0 +1,41 @@
+import { expose } from 'comlink';
```

Comment:
This is the worker. It exposes a method for everything we workerise, but it loads the bulk of the code lazily.

Reply:
I like this a lot better than the per-encoder approach.